Require.load not found in cache

I'm trying to build [this site](https://github.com/Cjen1/cjen1.github.io/blob/src/Site.hs), and I'm persistantly getting cache errors:

```
[ERROR] Hakyll.Core.Compiler.Require.load: papers/biblio.bib (snapshot _final) was not found in the cache, the cache might be corrupted or the item you are referring to might not exist
```

I think I have just missed something in the documentation, but I can't figure out why this isn't working... 

The relevant code is:

```
papersCompiler = do
  bib <- load "papers/biblio.bib"
  csl <- load "papers/acm_citation_style.csl"
  getResourceBody
    >>= readPandocBiblio defaultHakyllReaderOptions csl bib
    >>= return . writePandoc
```

My understanding is that this should try to find "papers/biblio.bib" in either the cache or the file system, but it doesn't seem to be checking the file system (which I've manually verified that its written correctly etc)?

Is there any file path rewriting going on which could be the issue?

--- a/lib/Hakyll/Web/Pandoc/Biblio.hs
+++ b/lib/Hakyll/Web/Pandoc/Biblio.hs
@@ -122,6 +122,7 @@ readPandocBiblio ropt csl biblio item = do
     zeroTime = Time.UTCTime (toEnum 0) 0
 
 --------------------------------------------------------------------------------
+-- | Compiles the markdown via Pandoc. Requires the csl and bib files to be known to the compiler via match statements.
 pandocBiblioCompiler :: String -> String -> Compiler (Item String)
 pandocBiblioCompiler cslFileName bibFileName = do
     csl <- load $ fromFilePath cslFileName

--- a/lib/Hakyll/Web/Pandoc/Biblio.hs
+++ b/lib/Hakyll/Web/Pandoc/Biblio.hs
@@ -122,7 +122,7 @@ readPandocBiblio ropt csl biblio item = do
     zeroTime = Time.UTCTime (toEnum 0) 0
 
 --------------------------------------------------------------------------------
--- | Compiles the markdown via Pandoc. Requires the csl and bib files to be known to the compiler via match statements.
+-- | Compiles a markdown file via Pandoc. Requires the .csl and .bib files to be known to the compiler via match statements.
 pandocBiblioCompiler :: String -> String -> Compiler (Item String)
 pandocBiblioCompiler cslFileName bibFileName = do
     csl <- load $ fromFilePath cslFileName

